dbus-codegen/src/main.rs

```diff
@@ -47,15 +47,23 @@ Defaults to 'RefClosure'."))
              .help("If present, skips a specific prefix for interface names, e g 'org.freedesktop.DBus.'."))
         .arg(clap::Arg::with_name("futures").short("f").long("futures")
              .help("Generates code to use with futures 0.3 (experimental)"))
+        .arg(clap::Arg::with_name("file").required(false).help("D-Bus XML Introspection file"))
         .get_matches();
 
+	if matches.value_of("destination").is_some() && matches.value_of("file").is_some() {
```

Comment:
Wrong indentation

Reply:
To clarify: Please use four spaces per indentation level (not tabs).

Reply:
we should setup `rustfmt` check in the CI (and CI itself) I think.

Reply:
Here is an example (although for gitlab): https://gitlab.gnome.org/GNOME/fractal/blob/master/.gitlab-ci.yml#L23